src/Pages/ContentListTable/components/SnapshotPackageModal/SnapshotPackageModal.test.tsx

```diff
@@ -76,3 +76,30 @@ it('Render with no packages (after an unsuccessful search)', () => {
   expect(queryByText('No packages match the search criteria')).toBeInTheDocument();
   expect(queryByText('Clear search')).toBeInTheDocument();
 });
+
+it('Clicks on package in "View All Snapshots" modal, opens new modal, and goes back to original repo', async () => {
+  (useGetPackagesQuery as jest.Mock).mockImplementation(() => ({
+    isLoading: false,
+    data: {
+      data: [packageItem],
+      meta: { count: 1, limit: 20, offset: 0 },
+    },
+  }));
+
+  const { getByText, findByText } = render(
+    <ReactQueryTestWrapper>
+      <SnapshotPackageModal />
+    </ReactQueryTestWrapper>,
+  );
+
+  fireEvent.click(getByText('Packages'));
+  expect(await findByText('View All Snapshots')).toBeInTheDocument();
```

Comment:
I'm a bit confused what this is doing.  You are rendering the modal, and then trying to find "Packages" which isn't in the modal and click on it?

Reply:
oh, i see, i think you are wanting to:

* Load the repositories list
* click view all snapshots
* click on the package count (opens the modal)
* click on something to close the modal
* verify you are back on the correct page.  

Is that right?

Reply:
Yeah

Reply:
Thats right